### DISCUSSION TITLE
第5节课作业(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#406] [作业]
第5课作业(24班)

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135683291

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/1677b00e-799b-44a4-a8ae-e05ed7967ea5

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29928434/

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day5_0.md

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135771342?spm=1001.2014.3001.5501

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135782784

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679371266

**Reply by Xinyi-Cheng** [reply]
更正：https://zhuanlan.zhihu.com/p/679798167

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135834479?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135844200?spm=1001.2014.3001.5501
